remote_config/testapp/Assets/Firebase/Sample/RemoteConfig/UIHandlerAutomated.cs

```diff
@@ -13,6 +15,11 @@ public class UIHandlerAutomated : UIHandler {
       Func<Task>[] tests = {
         TestDisplayData,
         TestDisplayAllKeys,
+// Skip the Realtime RC test on desktop as it is not yet supported.
+#if (UNITY_IOS || UNITY_TVOS || UNITY_ANDROID) && !UNITY_EDITOR
+        TestAddOnConfigUpdateListener,
+#endif  // !(UNITY_IOS || UNITY_TVOS || UNITY_ANDROID) || UNITY_EDITOR
+        TestAddAndRemoveConfigUpdateListener,
```

Comment:
this line should also be inside the #if right?

Reply:
Yea it probably should be. It technically passed on desktop since add and remove are no-op on desktop, but might as well not test those methods.